frontend/packages/console-shared/src/selectors/machine.ts

```diff
@@ -33,5 +33,5 @@ export const getMachineAddresses = (machine: MachineKind) => machine?.status?.ad
 
 export const getMachinePhase = (obj: MachineKind): string => {
   const phase = obj?.status?.phase;
-  return phase === 'Running' ? 'Provisioned as node' : phase;
+  return phase || 'Provisioned as node';
```

Comment:
We shouldn't return `Provisioned as node` if we don't have a phase.

```ts
return obj?.status?.phase;
```

If we change this, we can also remove the special machine sort method.

Reply:
> We shouldn't return `Provisioned as node` if we don't have a phase.
> 
> ```ts
> return obj?.status?.phase;
> ```
> 
> If we change this, we can also remove the special machine sort method.



> This is an intentional change in #4245. I'd say we simply close this as not a bug.
> 
> cc @rawagner @enxebre for context
> 
> /hold

@spadgett Thanks for the feedback. I have closed the bug.